SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -1018,16 +1024,23 @@ void FSpatialGDKEditorToolbarModule::OpenInspectorURL()
 	}
 }
 
-void FSpatialGDKEditorToolbarModule::LaunchInspectorWebpageButtonClicked()
+void FSpatialGDKEditorToolbarModule::StartInspectorProcess(TFunction<void()> InOnReady)
 {
+	auto OnReady = [InOnReady]() {
+		if (InOnReady)
+		{
+			InOnReady();
+		}
+	};
+
 	const USpatialGDKEditorSettings* SpatialGDKEditorSettings = GetDefault<USpatialGDKEditorSettings>();
 	const FString InspectorVersion = SpatialGDKEditorSettings->GetInspectorVersion();
 
-	AsyncTask(ENamedThreads::AnyBackgroundThreadNormalTask, [this, InspectorVersion] {
+	AsyncTask(ENamedThreads::AnyBackgroundThreadNormalTask, [this, InspectorVersion, OnReady] {
 		if (InspectorProcess && InspectorProcess->Update())
 		{
-			// We already have an inspector running. Just open the URL.
-			OpenInspectorURL();
+			// We already have an inspector process running. Do nothing.
```

Comment:
We do not do nothing.
```suggestion
			// We already have an inspector process running. Call ready callback if any.
```